pulp_2to3_migration/app/migration.py

```diff
@@ -17,14 +17,15 @@
 _logger = logging.getLogger(__name__)
 
 
-async def migrate_content(plugins_to_migrate):
```

Comment:
It'll be better to just pass in the plan everywhere for consistency, and then the functions can use whatever parts of the plan they need.  The info is pre-processed and cached so there is no cost to doing so.